src/System.Security.Cryptography.Encoding/src/Internal/Cryptography/OidLookup.OSX.cs

```diff
@@ -53,7 +52,7 @@ private static string NativeFriendlyNameToOid(string friendlyName, OidGroup oidG
         // and OpenSSL produce different answers.  Since the answers originally came from OpenSSL
         // on macOS, this preserves the OpenSSL names.
         private static readonly Dictionary<string, string> s_extraFriendlyNameToOid =
-            new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase)
+            new Dictionary<string, string>(8, StringComparer.OrdinalIgnoreCase)
```

Comment:
where does "8" come from?

Reply:
The number of items being added immediately after this line.  This is just pre-sizing it.